kolibri/plugins/learn/assets/src/views/ChannelsPage.vue

```diff
@@ -41,10 +41,19 @@
     },
     methods: {
       genChannelLink(channel_id) {
-        return {
-          name: PageNames.TOPICS_CHANNEL,
-          params: { channel_id },
-        };
+        let currentChannel = this.channels.find(channel => channel.id == channel_id);
+        if (currentChannel && currentChannel.options.modality == 'CUSTOM_NAVIGATION') {
```

Comment:
We like using strict equality, since it can root out certain types of bugs.

```suggestion
        let currentChannel = this.channels.find(channel => channel.id === channel_id);
        if (currentChannel && currentChannel.options.modality === 'CUSTOM_NAVIGATION') {
```